interface/forms/fee_sheet/new.php

```diff
@@ -28,6 +28,14 @@
 require_once("$srcdir/FeeSheetHtml.class.php");
 require_once("codes.php");
 
+//acl check
+$can_view = acl_check('admin', 'superbill', false, 'view');
+$can_write = acl_check('admin', 'superbill', false, 'write');
+if (!$can_view && !$can_write)
+{
+  formJump();
+}
+
```

Comment:
@matrix-amiel ,
This is not correct. I should of caught this on review. The admin superbill acl is for the administration->codes sheet and should not be used here. This form is already controlled by the encounters->codes and that is what should be used. Also not a fan of this formJump since it's silent; much better to show a message that not authorized. I am going to submit a fix PR for this in near future.